package-lock.json

```diff
@@ -60,7 +60,7 @@
     "@types/babel-generator": {
       "version": "6.25.1",
       "resolved": "https://registry.npmjs.org/@types/babel-generator/-/babel-generator-6.25.1.tgz",
-      "integrity": "sha512-nKNz9Ch4WP2TFZjQROhxqqS2SCk0OoDzGazJI6S+2sGgW9P7N4o3vluZAXFuPEnRqtz2A0vrrkK3tjQktxIlRw==",
+      "integrity": "sha1-+GqzzxMrBFl/5sQx0wg6rxt2tTA=",
```

Comment:
Just curious if this should be sha512 instead.

Reply:
This file is auto-generated/updated by npm.